include/wx/gtk/mimetype.h

```diff
@@ -23,7 +23,11 @@
 class WXDLLIMPEXP_CORE wxGTKMimeTypesManagerImpl : public wxMimeTypesManagerImpl
 {
 protected:
+#if defined(__UNIX__)
     wxString GetIconFromMimeType(const wxString& mime) wxOVERRIDE;
+#else
+    wxString GetIconFromMimeType(const wxString& mime);
```

Comment:
Why do we need it in non-Unix case? Is it going to be used at all? Under Unix it's used as a hook for the GUI toolkit to provide access to GUI-specific functionality, but this doesn't seem to make any sense under MSW...